Tailviewer/BusinessLogic/ActionCenter/IOpenFolderAction.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Tailviewer.BusinessLogic.ActionCenter
+{
+	public interface IOpenFolderAction 
+		: INotification
+	{
+		Exception Exception { get; }

```

Comment:
This interface is to be used by a view model eventually so I think having a message and an is failed property is more appropriate, since exception messages are usually horrible and translated (whereas Tailviewer itself is english only).

Reply:
Not sure I got this. Need to discuss.

Reply:
I thought about removing the exception property because it carries a (possibly) translated message which therefore shouldn't be shown in the UI directly. However maybe an additional error message would be appropriate? Or truly wrap the Task in a "FolderCannotBeOpenedException"?